2022q2/clusteradm.adoc

```diff
@@ -0,0 +1,30 @@
+=== Cluster Administration Team
+
+Links: +
+link:https://www.freebsd.org/administration/#t-clusteradm[Cluster Administration Team members] URL: link:https://www.freebsd.org/administration/#t-clusteradm[https://www.freebsd.org/administration/#t-clusteradm]
+
+Contact: Cluster Administration Team <clusteradm@FreeBSD.org>
+
+FreeBSD Cluster Administration Team members are responsible for managing the machines the Project relies on to synchronise its distributed work and communications.
+In this quarter, the team has worked on the following:
+
+* Installed a new mirror in Sydney, Australia hosted by IX Australia
+* Fixed CI cluster hardware failure
+* Set up a new internal monitoring system
+* Regular cluster-wide software upgrades
+* Regular support for FreeBSD.org user accounts
+
+Work in progress:
+
+* Work with the PowerPC team to improve the package builders, universal, and reference machines.
+* Plan Hardware refresh, and fixing misc failures in each sites
+* Improve the package building infrastructure
+* Review the service jails and service administrators operation
+* Working with doceng@ to improve https://www.freebsd.org and https://docs.freebsd.org
```

Comment:
Clarify: improve contents? Improve hosting performance? Something else?

Reply:
Improvements to the continuous deployment infrastructure driving the publicly visible builds of www.f.o and docs.f.o.

Reply:
> Improvements to the continuous deployment infrastructure driving the publicly visible builds of [www.f.o](http://www.f.o) and docs.f.o.

Thanks. In the interest of not delaying the status reports, do you mind if I merge and s/improve/improve deployment of/?

Reply:
Please do.  Thank you!